ts/docs.ts

```diff
@@ -23,9 +23,47 @@ import {
 } from './types'
 import { isArticle } from './util/guards'
 
+// counts <Type>.schema.yaml -> <Type>.html
+// and number of thses files that were injected w/ examples
+// note... htmlFromYamls >= examplesInserted (count is per 1 insert op)
+const stats = {
+  htmlFromYamls: 0,
+  examplesInserted: 0
+}
+
 // eslint-disable-next-line @typescript-eslint/no-floating-promises
 docs()
 
+/**
+ * Return array of examples (unparsed yaml string) for given
+ * schemaTitle. Looks for `examples/<schemaTitle>/*.<schemaTitle>.yaml`.
+ *
+ * path-matching w/ globby is not very good yet because some of the examples seem to
+ * have idiosyncratic names in their path.
+ *
+ * aplogies for this terrible typescript, I am not very familiar and just trying
+ * to demonstrate the concept. don't use this for real w/o fixing types.
```

Comment:
Please remove this comment, it's totally unnecessary since you did a great job and we're very grateful for your contributions :) Thank you!

I've left comments/suggestions with the changes for the type definitions, and feel free to reach out to me if you ever have any questions, either on GitHub or on [Zulip](http://stencila.zulipchat.com)